fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/ambassador/SnsAmbassador.java

```diff
@@ -151,18 +147,6 @@ private void process(TrafficLightRegistration interaction) {
         }
     }
 
-    private void process(VehicleRegistration interaction) {
-        final VehicleMapping applicationVeh = interaction.getMapping();
```

Comment:
This code was not required, as for all vehicles which send an AdhocCommunicationConfiguration an internal node is created.